tembo-olap/run.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+TRIES=3
+
+cat queries.sql | while read query; do
+    echo "$query";
```

Comment:
Like in other benchmarks, would recommend to drop cache after every query in order to make sure that query isn't being served from linux page cache.

Reply:
Hi @chhetripradeep ,

Reply:
Sounds good.